packages/wrangler/src/cloudchamber/deploy.ts

```diff
@@ -0,0 +1,168 @@
+import { readFileSync, statSync } from "fs";
```

Comment:
this doesn't seem to be tested at all?

Reply:
I didn't realise the test you changed had `containers` configured in the config file